requirements/dev.in

```diff
@@ -1,11 +1,11 @@
 # Additional requirements for development of this application
 
-diff-cover                # Changeset diff test coverage
-edx-lint                  # For updating pylintrc
+diff-cover==0.9.8         # Changeset diff test coverage
```

Comment:
hmm.. this is counter to the recommendation in [OEP-18](https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html).  Is this really the tech debt we want to take on?  Did you try just updating the minimal dependencies required for edx-drf-extensions?

FYI @jmbowman and @georgebabey - `make upgrade` doesn't work in this repo.  It complains with the following output and we can't figure out what dependency is requiring `bleach>=2.1.0`:

```
Could not find a version that matches bleach==1.4,>=2.1.0
Tried: 0.2.1, 0.2.1, 0.2.2, 0.2.2, 0.3, 0.3, 0.3.1, 0.3.1, 0.3.3, 0.3.3, 0.3.4, 0.3.4, 0.5.0, 0.5.0, 0.5.1, 0.5.1, 1.0.0, 1.0.0, 1.0.1, 1.0.1, 1.0.2, 1.0.2, 1.0.3, 1.0.3, 1.0.4, 1.0.4, 1.1.0, 1.1.0, 1.1.1, 1.1.1, 1.1.2, 1.1.2, 1.1.3, 1.1.3, 1.1.4, 1.1.4, 1.1.5, 1.1.5, 1.2, 1.2, 1.2.1, 1.2.1, 1.2.2, 1.2.2, 1.4, 1.4, 1.4.1, 1.4.1, 1.4.2, 1.4.2, 1.4.2, 1.4.2, 1.4.3, 1.4.3, 1.4.3, 1.4.3, 1.5.0, 1.5.0, 1.5.0, 1.5.0, 2.0.0, 2.0.0, 2.0.0, 2.0.0, 2.1, 2.1, 2.1, 2.1, 2.1.1, 2.1.1, 2.1.1, 2.1.1, 2.1.2, 2.1.2, 2.1.2, 2.1.2, 2.1.3, 2.1.3, 2.1.3, 2.1.3, 2.1.4, 2.1.4, 2.1.4, 2.1.4, 3.0.0, 3.0.0, 3.0.0, 3.0.0, 3.0.1, 3.0.1, 3.0.1, 3.0.1, 3.0.2, 3.0.2, 3.0.2, 3.0.2
There are incompatible versions in the resolved dependencies.
```

Reply:
I could not find what was the cause of the bleach issue so it was simpler to start with something that worked. You can work from either direction. Now that I have something that works, I can slowly unpin again until it breaks to find out what was requesting that version of bleach, since I couldn’t determine it using pipdeptree.

Reply:
Thanks @jmbowman.  I'll see if I can get a passing build with fewer pins.